doc/dev_guide/internals/iproto/requests.rst

```diff
@@ -38,6 +38,10 @@ Overview
             -   0x8XXX |br| MP_INT
             -   Error response
 
+        *   -   :ref:`IPROTO_UNKNOWN <internals-iproto-keys-unknown>`
+            -   -1 |br| MP_UINT
+            -   Unknown request type. The constant is used only for IPROTO handler :ref:`overriding <reference_lua-box_iproto_override>`.
```

Comment:
Your description is not so precise, I would rather refer to the `internals-iproto-unknown` section below.